### PR TITLE
Make scripts to work from any directory

### PR DESCRIPTION
**Pull request by lzap**
Solved https://github.com/RHEnVision/provisioning-backend/pull/54#pullrequestreview-1012697709

### CODE REVIEW

File: scripts/sources.clean.sh [hunk 1]
```diff
@@ -2,7 +2,8 @@
 #
 # See README.sources
 #
-source sources.conf
+BASEDIR=$(dirname $0)
+source $BASEDIR/sources.conf
 rm -rf ./sources-api-go ./sources-database-populator/
```
[the remark this answers is not in the document]
Comment (lzap) [reply scripts/sources.clean.sh:7]: Damn, busted! (I did not test this yeah.)

**Comment by lzap** [comment]
Bout' now.

File: scripts/sources.seed.sh [hunk 2]
```diff
@@ -2,8 +2,9 @@
 #
 # See README.sources
 #
-source sources.conf
-git clone https://github.com/MikelAlejoBR/sources-database-populator
-pushd sources-database-populator
+BASEDIR=$(dirname $0)
+source $BASEDIR/sources.conf
+git clone https://github.com/MikelAlejoBR/sources-database-populator $BASEDIR
```
Comment (ezr-ondrej) [inline scripts/sources.seed.sh:7]:
```suggestion
git clone https://github.com/MikelAlejoBR/sources-database-populator $BASEDIR/sources-database-populator
```

File: scripts/sources.setup.sh [hunk 3]
```diff
@@ -2,8 +2,9 @@
 #
 # See README.sources
 #
-source sources.conf
-git clone https://github.com/RedHatInsights/sources-api-go
+BASEDIR=$(dirname $0)
+source $BASEDIR/sources.conf
+git clone https://github.com/RedHatInsights/sources-api-go $BASEDIR
```
Comment (ezr-ondrej) [inline scripts/sources.setup.sh:7]:
```suggestion
git clone https://github.com/RedHatInsights/sources-api-go $BASEDIR/sources-api-go
```

**Review by ezr-ondrej — COMMENTED** [review]
It's cloning to the /scripts directly (I also thought it works like this, luckily I've tested it ^_^)

**Comment by lzap** [comment]
Sorry try now.

**Review by ezr-ondrej — APPROVED** [review]
Thanks @lzap ! nice improvement :+1:

**Pull request merged**